events_structs.go

```diff
@@ -967,6 +1080,18 @@ func (pev *Event) Specialize() DetailedEvent {
 		e = &EventLinkGithubType{Event: ev}
 	case EventUnlinkGithub:
 		e = &EventUnlinkGithubType{Event: ev}
+	case EventLoginSuccess:
```

Comment:
Shouldn't you add the DB related events in this switch / case (e.g. EventDatabaseAddFeature)?

Reply:
You should "Resolve" the conversations. The person who opened it is responsible for considering it closed.

In this case I don't see where you fixed this comment? This switch / case statement must contain the `EventDatabase*Feature` cases.